traffic_ops/traffic_ops_golang/wrappers.go

```diff
@@ -186,7 +186,7 @@ func gzipIfAccepts(r *http.Request, w http.ResponseWriter, b []byte) ([]byte, er
 	if len(b) == 0 || !acceptsGzip(r) {
 		return b, nil
 	}
-	w.Header().Set(api.ContentType, api.Gzip)
+	w.Header().Set("Content-Encoding", api.Gzip)
```

Comment:
What's this little guy about? Seems unrelated to the rest of the PR.

Reply:
Fixes https://github.com/apache/incubator-trafficcontrol/issues/1354
I assume this was an accidental commit, added while debugging to work around the bug in master. Needs to be reverted, because there's already a PR fixing which will cause conflicts https://github.com/apache/incubator-trafficcontrol/pull/1359

Reply:
this was a bug we found while testing.  My ContentType header of "application/json" was being overwritten by this setting and the son was being returned gzipped.  Rob spotted this and wrote a bug up on it as the content encoding and not the type should be gzip.